data.json

```diff
@@ -1,2215 +1,1742 @@
-            "label": "good first issue",
-            "technologies": [
-                "JavaScript",
-                "TypeScript"
-            ],
-            "description": "Next generation frontend tooling. It's fast! Alternative to Create React App"
-        },
-        {
-            "name": "Manifest",
-            "link": "https://github.com/mnfst/manifest",
-            "label": "good first issue",
-            "technologies": ["TypeScript"],
-            "description": "Manifestis an open-source Backend-as-a-Service allowign developers to create a backend easily and quickly."
-        },
-        {
-            "name": "grommet",
-            "link": "https://github.com/grommet/grommet",
-            "label": "good first issue",
-            "technologies": [
-                "JavaScript"
-            ],
-            "description": "a react-based framework that provides accessibility, modularity, responsiveness, and theming in a tidy package"
-        },
-        {
-            "name": "Rawsec Cybersecurity Inventory",
-            "link": "https://gitlab.com/rawsec/rawsec-cybersecurity-list",
-            "label": "difficulty::easy",
-            "technologies": [
-                "JavaScript",
-                "JSON",
-	        "Pug"
-            ],
-            "description": "An inventory of tools and resources that aims to help people to find everything related to CyberSecurity."
-        },
-        {
-            "name": "zoom-rs",
-            "link": "https://github.com/security-union/zoom-rs",
-            "label": "good first issue",
-            "technologies": [
-                "Rust"
-            ],
-            "description": "Teleconference system with a web based user interface written in Rust"
-	    },
-        {
-            "name": "FastAPI",
-            "link": "https://github.com/tiangolo/fastapi",
-            "label": "good first issue",
-            "technologies": [
-                "Python"
-            ],
-            "description": "A modern, fast (high-performance) web framework for building APIs with Python 3.6+ based on standard Python type hints."
-        }
-    ]
+  "sponsors": [
+    {
+      "name": "Michał Gołkowski",
+      "image": "https://avatars.githubusercontent.com/u/40803091",
+      "link": "https://github.com/MixeroTN"
+    }
+  ],
+  "technologies": {
+    ".NET": "net",
+    "C": "c-0",
+    "C#": "c-1",
+    "C++": "c-2",
+    "Java": "c-3",
+    "JavaScript": "c-4",
+    "TypeScript": "c-4",
+    "Go": "c-5",
+    "Python": "c-6",
+    "Angular": "c-7",
+    "MLOps": "c-8",
+    "Haskell": "c-9",
+    "JSON": "c-10",
+    "Pug": "c-11",
+    "Clojure": "c-12",
+    "Elixir": "c-13",
+    "Dart": "c-14",
+    "Kotlin": "c-15",
+    "Perl": "c-16",
+    "PHP": "c-17",
+    "Julia": "c-18",
+    "RUST": "c-19",
+    "Ruby": "c-20",
+    "Scala": "c-21",
+    "Smalltalk": "c-22",
+    "Swift": "c-23",
+    "Elm": "c-24",
+    "Ansible": "c-25"
+  },
```

Comment:
This looks like it was AI generated and just followed a pattern that seemed reasonable.